src/Momento.Sdk/Responses/CacheDeleteResponse.cs

```diff
@@ -23,5 +23,9 @@ public MomentoErrorCode ErrorCode
         {
             get => _error.ErrorCode;
         }
+
+        public string Message {
+            get => $"{_error.MessageWrapper}: {_error.Message}";
```

Comment:
oh, I see - so with this pattern we would always be surfacing both your wrapper message and the one from the server?

Reply:
Yes, it includes the static preamble for context and then the specific exception message coming from the server (or validators or whatever).